spec/index.bs

```diff
@@ -15194,10 +15194,9 @@ Rasterization produces a list of <dfn dfn>RasterizationPoint</dfn>s, each contai
 
             depth(|n|) = |vp|.`minDepth` &plus; |n|.`z` &times; ( |vp|.`maxDepth` - |vp|.`minDepth` )
 
-    1. Let |rasterizationPoints| be an empty list.
-
-        <p class="note editorial"><span class=marker>Editorial note:</span> specify that each rasterization point gets assigned an interpolated `divisor(p)`,
-        `framebufferCoords(n)`, `depth(n)`, as well as the other attributes.
+    1. Let |rasterizationPoints| be the list of points, each having its attributes (`divisor(p)`,
+        `framebufferCoords(n)`, `depth(n)`, etc.) interpolated according to its position on the
+        primitive, using the same interpolation as [[#primitive-clipping]].
```

Comment:
Are there user-defined attributes in there? If so their `@interpolate` mode is also a factor in how they're interpolated.

Reply:
Good call, updated.

Reply:
Ah, didn't realize for some reason that [[#primitive-clipping]] would have already explained this. Anyway this is clearer.